npm-shrinkwrap.json

```diff
@@ -10146,6 +10146,14 @@
           "resolved": "https://registry.npmjs.org/ansi-regex/-/ansi-regex-3.0.1.tgz",
           "integrity": "sha512-+O9Jct8wf++lXxxFc4hc8LsjaSq0HFzzL7cVsw8pRDIPdjKD2mT4ytDZlLuSBZ4cLKZFXIrMGO7DbQCtMJJMKw=="
         },
+        "cli-table": {
```

Comment:
hmm why did this file change?

Reply:
For me, this and the colours bit below get removed when I run `npm install` so not sure why this got added here. I repeated that 2-3 times and it got removed for me